lib/node_modules/@stdlib/array/base/slice/docs/types/index.d.ts

```diff
@@ -211,28 +57,16 @@ declare function slice( x: Complex128Array, start: number, end: number ): Comple
 * @returns output array
 *
 * @example
-* var Complex64Array = require( '@stdlib/array/complex64' );
-*
-* var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+* var x = [ 1, 2, 3 ];
 *
 * var out = slice( x, 0, 3 );
-* // returns <Complex64Array>[ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ]
-*/
-declare function slice( x: Complex64Array, start: number, end: number ): Complex64Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
+* // returns [ 1, 2, 3 ]
 *
 * @example
 * var x = [ 1, 2, 3, 4, 5, 6 ];
 *
-* var out = slice( x, 0, 3 );
-* // returns [ 1, 2, 3 ]
+* var out = slice( x, 0, 2 );
+* // returns [ 1, 2 ]
 */
 declare function slice<T = unknown>( x: Collection<T>, start: number, end: number ): Array<T>;
```

Comment:
Would it make sense to use the `interface` approach I mentioned previously, or should we be content with typing like this? Atm, this will return incorrect results for specialized constructors having a `slice` method which are not captured by `TypedArray | ComplexTypedArray`. Granted, the previous declarations also did not capture this case, but we could go ahead and correct here.

Reply:
@kgryte We should merge this PR without the proposed change as I am not getting it to work properly with tuple types. Have some work-in-progress stuff we can discuss but this is quite complicated and would also require some opaque typing, I believe.

Reply:
Okay. We should create a TODO to follow-up.